src/client/src/shell/GlobalScrollbarStyle.tsx

```diff
@@ -10,6 +10,9 @@ export const css = memoize(
     body ::-webkit-scrollbar-thumb {
       background-color: ${color};
     }
+    body {
+      scrollbar-color: ${color} transparent;
```

Comment:
Adding `scrollbar-width` to `body` has no affect, so I had to add these specifically to our scrollable panels. I left colour here to avoid recreating the `rgba` value in each file.